packages/venia-ui/lib/components/Gallery/item.js

```diff
@@ -1,138 +1,113 @@
-import React, { Component } from 'react';
+import React from 'react';
 import { string, number, shape } from 'prop-types';
 import { Link, resourceUrl } from '@magento/venia-drivers';
 import { Price } from '@magento/peregrine';
-
-import classify from '../../classify';
 import { transparentPlaceholder } from '@magento/peregrine/lib/util/images';
-import { generateSrcset } from '../../util/images';
+
+import { mergeClasses } from '../../classify';
+import Image from '../Image';
 import defaultClasses from './item.css';
 
 // The placeholder image is 4:5, so we should make sure to size our product
 // images appropriately.
 const imageWidth = '300';
```

Comment:
```suggestion
const IMAGE_WIDTH = '300';
```

Reply:
Resolved in https://github.com/magento/pwa-studio/pull/1871/commits/97b24bc8d601f52d93d26756bdb9154d02ba6170.